### ISSUE TITLE
Error in installation: unable to collate and parse R files for package

### ISSUE THREAD

**Issue by bsvisser** [grimbough/FITfileR#22]
I'm trying to install the package to use a script from someone else, but I'm getting an error in installation. I don't have a lot of knowledge of R, so excuse me if this issue doesn't have to do with the package (although it seems to).

```R
* installing *source* package 'FITfileR' ...
** using staged installation
** R
Error in parse(outFile) : 
  E:/TEMP/Rtmpkl0MSI/R.INSTALL91442106a47/FITfileR/R/read_headers.R:70:25: unexpected '>'
69: isDefinition <- function(header) {
70:   rawToBits(header)[7] |>
                            ^
ERROR: unable to collate and parse R files for package 'FITfileR'
* removing 'C:/Users/Brent Visser/Documents/R/win-library/4.0/FITfileR'
Warning message:
In i.p(...) :
  installation of package ‘E:/TEMP/RtmpwbVqhx/file29a0399efcd/FITfileR_0.1.5.tar.gz’ had non-zero exit status
```

How can I work around this?

**Comment by grimbough** [comment]
I think this will be resolved if you use R 4.1 or greater. The pipe operator `|>` was only added in that version.

I should make this explicit in the package DESCRIPTION file, or move away from using the pipe internally.

**Comment by bsvisser** [comment]
Unsurprisingly, that fixed it! Thanks for the quick help.
